docs/content/en/docs/reference/support/support-versions.md

```diff
@@ -61,6 +61,13 @@ You can find the content of the previous Amazon EKS Anywhere minor versions and
 ### Will I get notified when there is a new Amazon EKS Anywhere version release?
 You will get notified if you have subscribed as documented on the [Release Alerts]({{< relref "../snsupdates/" >}}) page.
 
+### Can I use different Amazon EKS Anywhere minor versions for my management cluster and workload clusters?
+
+Yes, the management cluster and its workload clusters can be created using different Amazon EKS Anywhere minor versions.
```

Comment:
```suggestion
Yes, the management cluster can be upgraded to newer Amazon EKS Anywhere versions than its workload clusters.
```

Technically you can't create a workload cluster on a version if the management cluster was never in that version. Example: create management cluster with v0.16.0. You can create workload clusters with v0.16.0 but not with v0.15.0. On the contrary, if you create your management cluster with version v0.15.0 and then upgrade to v0.16.0, you can create workload clusters in either v0.16.0 or v0.15.0.

Not sure if we want to add all this detail

Reply:
I definitely can see this coming up as a question from users, so I added a note with those details